packages/web/hooks/ui-config/use-add-concentrated-liquidity-config.ts

```diff
@@ -98,5 +101,33 @@ export function useAddConcentratedLiquidityConfig(
     config.poolId,
   ]);
 
-  return { config, addLiquidity };
+  const increaseLiquidity = useCallback(
+    (positionId: string) => {
+      return new Promise<void>(async (resolve, reject) => {
+        const amount0 = config.baseDepositAmountIn.getAmountPrimitive().amount;
+        const amount1 = config.quoteDepositAmountIn.getAmountPrimitive().amount;
+
+        await account.osmosis.sendAddToConcentratedLiquidityPositionMsg(
+          positionId,
+          amount0,
+          amount1,
+          undefined,
+          undefined,
+          (tx) => {
+            if (tx.code) reject(tx.log);
+            resolve();
+          }
+        );
+
+        try {
+        } catch (e: any) {
+          console.error(e);
```

Comment:
Did you mean to leave this in?

Reply:
The console log? Another option would be to remove that and have the caller of `increaseLiquidity` handle the error as they please? Fine with either.

Reply:
The latter makes more sense to me, but I don't have the full context